Cargo.toml

```diff
@@ -44,4 +45,4 @@ default = ["tls"]
 # enable tls
 tls = ["hyper-tls"]
 # enable etag-based http_cache functionality
-httpcache = []
\ No newline at end of file
+httpcache = []
```

Comment:
You could make dirs an optional dependency and enable it here. This would make it 0 cost when this feature is disabled

Reply:
I'm happy to do so, but why would someone disable it?  The home_dir in the standard library is deprecated because it's broken.  Maybe I'm missing something but I think it's a tiny crate whose cost is negligible, no?

Reply:
Done.  Rate my gating skills 🙂